scripts/e2e-tests.sh

```diff
@@ -127,6 +127,21 @@ function save_metadata() {
 EOF
 }
 
+# Returns the latest gke version in [major].[minor].[patch] with given [major](.[minor])? Kubernetes version
+# Parameters: $1 - prefix of gke versions
+#             $2 - gcp zone
+function get_latest_gke_version() {
+  local version_prefix="$1"
```

Comment:
You can replace these five commands with a single one and inline it:

```
$ gcloud container get-server-config --zone=us-central1 --format="value(validMasterVersions)" 2> /dev/null | tr ';' '\n' | grep -F 1.13 | cut -f1 -d- | sort | tail -1
1.13.6
```

Reply:
done